app/style/fonts/zeta-neue.css

```diff
@@ -267,21 +213,12 @@
 .icon-hourglass:before {
   content: "\e239";
 }
-.icon-collection:before {
-  content: "\e260";
-}
 .icon-av:before {
   content: "\e261";
 }
 .icon-dismiss:before {
   content: "\e262";
 }
-.icon-checkbox:before {
-  content: "\e600";
-}
-.icon-checkbox-checked:before {
-  content: "\e601";
-}
 .icon-contacts:before {
   content: "\e602";
 }
```

Comment:
We shouldn't really manually edit this file. Design creates it as the output when we generated a new icon font. How much work left to remove it completely?

Reply:
This file won't see any automatic updates anymore. All new icons now come as svg. Removing the obsolete ones is a good way of keeping track of missing svg implementations and to-be-replaced icons.

Reply:
I don't fully agree but let's go with it. I see removing the icon font as one task that will be done at the end if one chance of breaking stuff. Now we possibly break things everytime we add a new SVG based icon and forget about an instance of the old icon font based one.